view/widget/nbp_api_view_frame.py

```
@@ -0,0 +1,48 @@
+from service.log.nbp_service import NbpService
+from view.model.button import Button
+from view.model.label import Label
+from view.model.viewframe import ViewFrame
+from view.widget.logger import Logger
+from view.model.row import Row
+
+
+class NbpApiView(ViewFrame):
+
+    def __init__(self, parent):
```

Ruff

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)